aiosmtpd/tests/test_handlers.py

```diff
@@ -3,21 +3,24 @@
 
 import logging
 import sys
+from email.message import Message as Em_Message
 from io import StringIO
 from mailbox import Maildir
 from operator import itemgetter
 from pathlib import Path
 from smtplib import SMTPDataError, SMTPRecipientsRefused
 from textwrap import dedent
 from types import SimpleNamespace
-from typing import AnyStr, Generator, Type, TypeVar, Union
+from typing import AnyStr, Callable, Generator, Type, TypeVar, Union
 
 import pytest
 
 from aiosmtpd.controller import Controller
 from aiosmtpd.handlers import AsyncMessage, Debugging, Mailbox, Proxy, Sink
+from aiosmtpd.handlers import Message as AbstractMessageHandler
```

Comment:
Disambiguate with `email.message.Message` (which actually has been rename-imported to `Em_Message`, but let's just be very _very_ clear)